client/transaction-pool/src/maintainer.rs

```diff
@@ -74,6 +74,14 @@ where
 		id: &BlockId<Block>,
 		retracted: &[Block::Hash],
 	) -> Box<dyn Future<Output=()> + Send + Unpin> {
+		let now = std::time::Instant::now();
+		let took = move || {
+			let took = std::time::Instant::now().duration_since(now);
+			format!("Took {} ms", took.as_millis())
+		};
+
```

Comment:
Next time you can do something like
```
use tracing;
let span = tracing::span!(tracing::Level::DEBUG, "retracted");
let _enter = span.enter();
```
inside a context, to send the time to trace/grafana.

Reply:
Can I also display that in logs?

Reply:
Yeah, you need to use `--tracing-targets`  with an appropriated pattern (maybe `substrate_client`) and `--tracing-receiver` (but is defaulting to Log).